### ISSUE TITLE
Chaste fails to cmake on new Ubuntu 22.04 Jammy machine.

### ISSUE THREAD

**Issue by BJackal** [Chaste/Chaste#101]
Please see the [guides](https://chaste.cs.ox.ac.uk/trac/wiki/ChasteGuides), and [discussions](https://github.com/Chaste/Chaste/discussions) for answers to common problems.

**Describe the bug**
When using cmake for chaste on a new Ubuntu 22.04 machine after following wiki install guidelines you get the following error:

// BEGIN//

```
The imported target "vtkParseOGLExt" references the file
   "/usr/bin/vtkParseOGLExt-7.1"
but this file does not exist.  Possible reasons include:
* The file was deleted, renamed, or moved to another location.
* An install or uninstall procedure did not complete successfully.
* The installation package was faulty and contained
   "/usr/lib/cmake/vtk-7.1/VTKTargets.cmake"
but not all the files it references.

-- The imported target "vtkRenderingPythonTkWidgets" references the file
   "/usr/lib/x86_64-linux-gnu/libvtkRenderingPythonTkWidgets.so"
but this file does not exist.  Possible reasons include:
* The file was deleted, renamed, or moved to another location.
* An install or uninstall procedure did not complete successfully.
* The installation package was faulty and contained
   "/usr/lib/cmake/vtk-7.1/VTKTargets.cmake"
but not all the files it references.

-- The imported target "pvtk" references the file
   "/usr/bin/pvtk"
but this file does not exist.  Possible reasons include:
* The file was deleted, renamed, or moved to another location.
* An install or uninstall procedure did not complete successfully.
* The installation package was faulty and contained
   "/usr/lib/cmake/vtk-7.1/VTKTargets.cmake"
but not all the files it references.

-- VTK version: 7.1
-- No HDF5 found in /usr/lib/petscdir/3.15/: looking for alternative HDF5 instead
-- Could NOT find MPI_C (missing: MPI_C_WORKS) 
CMake Error at /usr/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find MPI (missing: MPI_C_FOUND) (found version "3.1")
Call Stack (most recent call first):
  /usr/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake-3.22/Modules/FindMPI.cmake:1830 (find_package_handle_standard_args)
  CMakeLists.txt:428 (find_package)
```

// END //

**To Reproduce**
The only way I am currently aware of reproducing this issue is attempting to install and then cmake chaste on a freshly built Ubuntu 22.04 system.

```
START FROM FRESH SETUP

sudo nano /etc/apt/sources.list.d/chaste.list
** deb [signed-by=/usr/share/keyrings/chaste.asc] http://www.cs.ox.ac.uk/chaste/ubuntu jammy/ **
sudo wget -O /usr/share/keyrings/chaste.asc https://www.cs.ox.ac.uk/chaste/ubuntu/Chaste%20Team.asc
sudo apt-get update
sudo apt-get install chaste-dependencies

git clone --recursive -b release https://chaste.cs.ox.ac.uk/git/chaste.git Chaste
sudo apt-get update
sudo apt-get install --install-recommends chaste-dependencies
sudo apt-get install `dpkg -s chaste-dependencies | egrep "^Suggests" | cut -d "," -f 1-111 --output-delimiter " " | cut -d ":" -f 2`

mkdir Chaste_build
cd Chaste_build
cmake &PATH TO CHASTE

**THE ABOVE ERROR WILL APPEAR**
```

**Expected behavior**
Cmake should work normally and then allow you to make with Chaste

**System**
Please provide relevant details of your system, such as:
 * OS version:: Ubuntu 22.04
 * CMake version:: 3.22.1
 * C++ compiler version:: gcc 11.3.0
 * Python version:: 3.10.6

```

**Additional context**
Thus far this has only been seen on 2 clean machines built with 22.04 and one machine updated from 20.04 to 22.04

**Comment by BJackal** [comment]
This issue can be resolved by merging the https://github.com/Chaste/Chaste/tree/fcooper8472-petsc branch into the develop branch locally on machines experiencing this issue.

**Comment by fcooper8472** [comment]
Fixed by #102
